core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/math/percentile.kt

```diff
@@ -0,0 +1,138 @@
+package org.jetbrains.kotlinx.dataframe.math
+
+import org.jetbrains.kotlinx.dataframe.impl.asList
+import java.math.BigDecimal
+import java.math.BigInteger
+import kotlin.reflect.KType
+import kotlin.reflect.typeOf
+
+public inline fun <reified T : Comparable<T>> Iterable<T>.percentileOrNull(percentile: Double): T? =
```

Comment:
I don't think DataFrame should have these functions public. It pollutes the scope on Iterables and it's outside the scope of DataFrame to offer these to users. They should be part of a statistics library if you want them publicly accessible :). (mentioned here https://github.com/Kotlin/dataframe/issues/961)

Reply:
yeah, that's true. Let's get rid of all of them In the other PR?